lesson18/src/main/java/com/lesson18/service/EmployeeService.java

```diff
@@ -0,0 +1,20 @@
+package com.lesson18.service;
+
+import com.lesson18.entity.Employee;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface EmployeeService {
+
+    void addEmployee(Employee employee);
+
+    void editEmployee(Employee employee, int employeeId);
+
+    void deleteEmployee(int employeeId);
+
+    Optional<Employee> find(int employeeId);
```

Comment:
In service, it should not be optional only in the repository layer.
What do you think?